testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/AbstractHttpServerTest.java

```diff
@@ -98,6 +98,7 @@ public abstract class AbstractHttpServerTest<SERVER> extends AbstractHttpServerU
   void setupOptions() {
     options.expectedServerSpanNameMapper = this::expectedServerSpanName;
     options.expectedHttpRoute = this::expectedHttpRoute;
+    options.setExpectedException(new IllegalStateException(EXCEPTION.getBody()));
```

Comment:
any reason you didn't change this in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/050e2999f3ff32151a102b6d49fcb7e6361b09bc/testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/HttpServerTestOptions.java#L43 instead?

Reply:
I missed this when looking for where it was configured, i'll update